game/configuration.cc

```diff
@@ -439,3 +494,17 @@ void populateBackends (const std::list<std::string>& backendList) {
 	backendConfig.selectEnum(selectedBackend);
 	backendConfig.setOldValue(backendConfig.getEnumName());
 }
+
+void populateLanguages(const std::map<std::string, std::string>& languages, bool refreshOutDated) {
+	ConfigItem& languageConfig = config["game/language"];
+	if (refreshOutDated) {
+		languageConfig.removeAllEnums();
+	}
+	for (auto const& language : languages) {
+		languageConfig.addEnum(language.second);
+	}
+	static std::string selectedLanguage = std::string();
```

Comment:
why is that static? BTW the initialization is useless.

Reply:
Removed the initialization and static